app/src/main/java/ch/epfl/sdp/artificial_intelligence/MovingEntity.java

```diff
@@ -38,11 +38,11 @@ public void setAcceleration(float acceleration) {
         this.acceleration = acceleration;
     }
 
-    public float getOrientation() {
+    public double getOrientation() {
```

Comment:
I see a lot of floats changed to double, for double the fun!

Reply:
Yeah, we may need to have this kind of precision here.